src/Java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialMixer.java

```diff
@@ -103,7 +103,7 @@ public String getCustomGUIResourceName() {
 
 	@Override
 	public GT_Recipe.GT_Recipe_Map getRecipeMap() {
-		return GT_Recipe.GT_Recipe_Map.sMixerRecipes;
+		return gregtech.api.util.GTPP_Recipe.GTPP_Recipe_Map.sMultiblockMixerRecipes_GT;
```

Comment:
Doesn't need to be the full class name here `gregtech.api.util.` is unnecessary.